### ISSUE TITLE
Имена расчетных таблиц

### ISSUE THREAD

**Issue by BlackWrangler** [GolosMovement/StatElectGenerator#99]
Если мы идём этим путём, то лучше давать такие имена, чтобы были в конце сортировки SQL manager. Чтобы не засоряли вид. Начинающиеся на Z? 

![image](https://user-images.githubusercontent.com/36114863/49042910-291d1c80-f1d2-11e8-9873-7bc51d9b9c0c.png)

**Comment by ybinzu** [comment]
Реализую в #98. Такое имя подойдет: `ZCalculateValues1234`?

**Comment by BlackWrangler** [comment]
Да. Пишут что можно _, @, #, но первые двое лезут вверх при сортировке. # - не дает создать.

Я буду экспериментировать с XCalcTbl1234
